File: src/main/java/test_lea_0003/a01_evenDigitsNumber/solution.rb
def evenDigitsNumber(a)
    return a.inject(0){|sum,x| sum + is_even(x) }
end

def is_even(number)
    if number < 10 or (number >= 100 and number < 1000)
        return 0
    end
    if number < 100 or (number >= 1000 and number < 10000)
        return 1
    end
    return 0
end

File: src/main/java/test_lea_0003/a01_evenDigitsNumber/solution.py
from typing import List


def even_digits_number(a: List[int]):
    return sum([is_even(x) for x in a])

def is_even(number: int):
    if number < 10 or (number >= 100 and number < 1000):
        return 0
    if number < 100 or (number >= 1000 and number < 10000):
        return 1
    return 0
